examples/rethinkdb/README.md

```diff
@@ -10,6 +10,12 @@ Setting up a [rethinkdb](http://rethinkdb.com/) cluster on [kubernetes](http://k
 
 Quick start
 -----------
+**Step 0**
+
+change the namespace of the current context to "rethinkdb"
+```
+$kubectl config view | grep "current-context" | awk '{print $2}' | xargs -I {} kubectl config set-context {} --namespace=rethinkdb
```

Comment:
We use 
```config view -o template --template='{{index . "current-context"}}'``` at other places like https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/kubectl.sh#L127 which seems cleaner.